modules/ROOT/pages/project-best-practices-index.adoc

```diff
@@ -1,33 +1,51 @@
 = Project best practices
-:description: Helpful experience we share for the good of your project.
-
-Helpful experience we share for the good of your project.
-
-blabla
-[.card-section]
-== Worth trying
-
-[.card.card-index]
---
-xref:teamwork.adoc[[.card-title]#Create a multi-skilled team# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:lifecycle-and-profiles.adoc[[.card-title]#Project phases# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:agility.adoc[[.card-title]#Agility# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:software-craftsmanship.adoc[[.card-title]#Software craftsmanship# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:devops.adoc[[.card-title]#DevOps# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
+:description: Helpful experience we share for the good of Bonita projects.
+
+Helpful experience we share for the good of Bonita projects.
```

Comment:
You declare a variable description juste above, use it instead of duplicating the content :) 

```suggestion
{description}
```

Reply:
I do not get that: this is how we do it on all doc pages.

Reply:
It is just to avoid duplication, I don't know how the pages were migrated, maybe the automatic migration did not handle this.